test/application_system_test_case.rb

```diff
@@ -1,4 +1,4 @@
-require "test_helper"
+require 'test_helper'
```

Comment:
Can't see any tests here :desert: 
You can fiddle with minitest here, but in real life :tm: we use RSpec